packages/react/src/input-base/InputBase.css.ts

```diff
@@ -82,11 +78,6 @@ export const input = recipe({
         },
       },
     }),
-    responsiveStyle({
```

Comment:
let's revert this change please - we can add a comment here on why this was done - but the idea here is to explicitly use `16px` on mobile because otherwise iOS will zoom in on the page anytime the input is focused

https://defensivecss.dev/tip/input-zoom-safari/

Reply:
Got it vai.